resources/uco_design_document.md

```diff
@@ -237,11 +297,21 @@ For instance, a digital photograph is represented as an ```observable:RasterPict
 }
 ```
 
-The generalized approach of facets on domain objects provides value in simple support for use of particular facets across multiple classes/objects, for simplified 3rd party extension of UCO classes/objects, and for clean granularity for adopting profiles or application domain extensions to choose which adornments they want to use.
-
-If particular characterizing properties are directly relevant to the object across most use cases they are typically defined as properties directly on the class/object but where they may be characterizing a particular aspect of an object relevant to some but not all use cases they are typically defined as a facet that can be applied to the class/object when appropriate.
-
-For example, the properties ```core:source``` and ```core:target``` are always relevant to the ```core:Relationship``` class/object in all use cases and are therefore defined as direct properties of ```core:relationship```. On the other hand, differing use cases may characterize a ```location:Location``` as a simple address, or as a set of latitude-longitude coordinates, or as a set of GPS coordinates or any combination thereof. Because of this variation, properties characterizing a location in the form of an address are defined using the ```location:SimpleAddressFacet```, properties characterizing a location in the form of a set of latitude-longitude coordinates are defined using the ```location:LatLongCoordinatesFacet```, and a location in the form of a set of GPS coordinates are defined using the ```location:GPSCoordinatesFacet```.
+The generalized approach of facets on domain objects provides value by mitigating the above limitations/downsides as well as supporting many of the necessary use cases.
+
+* Simple support for use of particular non-domain concept classes across multiple domain concept classes
+* Simplified 3rd party extension of UCO classes/objects
+* Reduces verbosity and complexity for appropriate use cases
+* Significantly reduces shape churn for appropriate cases
+* Supports duck-typing use case (described in #5.1 below) very simply

```

Comment:
> very simply

I warned @sbarnum on a call this morning that I will be excising "simply" from this Pull Request.  This was on a call, of which we plan to have several more of, where we were convened as part of the Facet Design Pattern Working Group to discuss the applicability, benefits, and detriments of `Facet`s.  Our efforts stand in contrast to "simple".

Besides that call, there was other editorial training I had in and since grad. school warning away from the phrase.  If `Facet`s are indeed simple, they must be so without having to declare them so.  I have personally found counter-examples to their simplicity, e.g. with complexities from is-a vs. has-a modeling (see e.g. [CASE-Examples Issue 97](https://github.com/casework/CASE-Examples/issues/97)) and incongruous capabilities (see e.g. the "cyborg" illustration in Problem 2 of [UCO Issue 438](https://github.com/ucoProject/UCO/issues/438)).